ci-operator/step-registry/cri-o/crun/e2e/test/cri-o-crun-e2e-test-commands.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
this can be done as a follow up, but is there anyway to share this bash code and replace variables? there's a decent amount of duplication

Reply:
Agree, I was thinking of handling that as part of the code optimization once we include all variants.